OWNERS_ALIASES

```diff
@@ -141,6 +141,9 @@ aliases:
   wg-policy-leads:
     - JimBugwadia
     - rficcaglia
+  wg-serving-leads:
+    - SergeyKanzhelev
+    - TBD
```

Comment:
How would the volunteering process be, I am more than happy to help here

Reply:
we can discuss at the first meeting. Did you have a chance to reply here: https://forms.gle/27mqbTC1xBP5QPSV9?